app/enterprise/1.3-x/deployment/access-license.md

```diff
@@ -24,14 +24,43 @@ Click into the **Files** section
 
 Click any file you would like to download.
 
-Alternatively, you can run this command in your terminal
+## Programmatically accessing your license file
+
+For programmatic access you'll need 3 pieces of information:
+
+ - username (was provided by email by your CSE)
+ - repository name (the Bintray repository, visible in the GUI, usually named after your company name)
+ - the Bintray API key for the account (see below)
+
+To get the API key follow these steps:
+
+1. [Log into Bintray](https://bintray.com/login?forwardedFrom=%2Fkong%2F)
+2. Open the Profile settings (click the username at the top-right and select "Edit Profile")
+3. On the left select "API Key", and provide your password again
```

Comment:
```suggestion
3. On the left select **API Key**, and if prompted, provide your password again. Click the copy icon to copy the key to your clipboard. 
```
Adding "if prompted" since I've never gotten the prompt - might be showing up cause of some security settings on your account?